ibm/service/iampolicy/resource_ibm_iam_access_management_account_settings.go

```diff
@@ -0,0 +1,291 @@
+												},
+											},
+										},
+									},
+									"service": {
+										Type:        schema.TypeSet,
+										Optional:    true,
+										MaxItems:    1,
+										Description: "The core set of properties associated with a service identity type.",
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"state": {
+													Type:        schema.TypeString,
+													Required:    true,
+													Description: "The state of the service identity type.",
+												},
+												"external_allowed_accounts": {
+													Type:        schema.TypeList,
+													Required:    true,
+													Description: "List of accounts that the state applies to for the service identity type.",
+													Elem:        &schema.Schema{Type: schema.TypeString},
+												},
+											},
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			"accept_language": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     "default",
+				Description: "Language code for translations* `default` - English* `de` -  German (Standard)* `en` - English* `es` - Spanish (Spain)* `fr` - French (Standard)* `it` - Italian (Standard)* `ja` - Japanese* `ko` - Korean* `pt-br` - Portuguese (Brazil)* `zh-cn` - Chinese (Simplified, PRC)* `zh-tw` - (Chinese, Taiwan).",
+			},
+			"account_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The account ID that the Access Management Account Settings belong to.",
+			},
+		},
+	}
+}
+
+func resourceIBMAccessManagementAccountSettingsGet(context context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	iamPolicyManagementClient, err := meta.(conns.ClientSession).IAMPolicyManagementV1API()
+	if err != nil {
+		tfErr := flex.TerraformErrorf(err, err.Error(), "ibm_iam_access_management_account_settings", "read")
+		log.Printf("[DEBUG]\n%s", tfErr.GetDebugMessage())
+		return tfErr.GetDiag()
+	}
+
+	var accountID string
+
+	if _, ok := d.GetOk("account_id"); ok {
+		accountID = d.Get("account_id").(string)
+	}
+
+	getSettingsOptions := &iampolicymanagementv1.GetSettingsOptions{
+		AccountID: &accountID,
+	}
+
+	if _, ok := d.GetOk("accept_language"); ok {
+		getSettingsOptions.SetAcceptLanguage(d.Get("accept_language").(string))
+	}
+
+	amAccountSettings, _, err := iamPolicyManagementClient.GetSettingsWithContext(context, getSettingsOptions)
+
+	if err != nil {
+		tfErr := flex.TerraformErrorf(err, fmt.Sprintf("GetSettingsWithContext failed: %s", err.Error()), "ibm_iam_access_management_account_settings", "read")
+		log.Printf("[DEBUG]\n%s", tfErr.GetDebugMessage())
+		return tfErr.GetDiag()
+	}
+	if _, ok := d.GetOk("external_account_identity_interaction"); ok {
+		d.Set("external_account_identity_interaction", flex.FlattenAMSettingsExternalIdentityInteraction(amAccountSettings))
+	}
+
+	d.SetId(fmt.Sprintf("amAccountSettings-%s", accountID))
```

Comment:
Why we want to append some constant "amAccountSettings"for setID. Can we set only to the accoutnID